packages/pigeon/run_tests.sh

```diff
@@ -88,6 +104,7 @@ pushd $PWD
 cd e2e_tests/test_objc/
 flutter pub get
 popd
+test_null_safe_dart ./pigeons/message.dart
```

Comment:
does it make sense to test the other scenarios?

Reply:
I did it when I was developing but then I realized I don't think any of the other cases are exercising the code differently.  The changes were in pieces of code that would have shown up in message.dart.